### DISCUSSION TITLE
Rejoining a room after getting disconnected stops updating own user panel for other players

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#15961] [Q&A] [answered]
When you get disconnected while in gameplay in multiplayer and rejoin the room, people will always see you as playing even when it's not the case, also found out now that stops updating mods too. 
Everything worked fine on my end, the mod which is talked about in chat wasn't turned on for me and i played without it, also didn't see myself with the "playing" status.

I have also seen it happen to another player a week ago.

(Image does not show the problem, but chat saying that it happened)
![image](https://user-images.githubusercontent.com/53081750/144912829-6a14d7a0-508f-445c-a92f-c2bd742e18a1.png)

Although they are not needed to confirm the issue above, here are the logs from my session, in case there's anything relevant. Disconnection is at 2021-12-06 19:24:12

[runtime.log](https://github.com/ppy/osu/files/7663237/runtime.log)
[database.log](https://github.com/ppy/osu/files/7663238/database.log)
[network.log](https://github.com/ppy/osu/files/7663234/network.log)
[performance.log](https://github.com/ppy/osu/files/7663236/performance.log)

**Comment by Sinamer** [comment]
Screenshot of the issue from my point of view:
![image](https://user-images.githubusercontent.com/53346335/144918549-dd31d624-c401-41f3-945c-ec37045f4c80.png)
Notice how the map is finished and other players are readying up for the next one, but user state still says "playing", and the Approach Different mod is enabled, even though the player confirmed it wasn't.

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #15970.
